### ISSUE TITLE
.travis.yml needs updating for offlineimap3

### ISSUE THREAD

**Issue by williamspatrick** [OfflineIMAP/offlineimap3#31]
#### General informations

The .travis.yml is unchanged from the original offlineimap fork and is specifying python2.  This should be updated to python3 and enabled on this repository.

**Comment by thekix** [comment]
Thanks a lot @williamspatrick 

I cannot update the Travis configuration. @nicolas33 could you create it for offlineimap3?

travis-ci.org is moving to travis-ci.org, probably both projects should be updated.

**Comment by nicolas33** [comment]
For travis CI to get working you need to log on travis-ci.com and import the project from there. You should have the rights to do so @thekix . If not, please let me know.

**Comment by nicolas33** [comment]
BTW, you're now a OfflineIMAP organization owner.

**Comment by thekix** [comment]
Hi @nicolas33 

I am trying to import the offlineimap/offlineimap3 project, but I cannot do it. After allow travis to access to github, if I enter in github, then settings, applications, travis-ci, I cannot select only this repository (it is not in the list).

I allowed travis for all my repos, then offlineimap3 is in the travis dashboard, with status "unknown", I tried to run it, but nothings happends.

If you want, write me by mail to make the setup.

Thanks a lot,
kix

**Comment by brandsimon** [comment]
Have you considered to use github workflow for this? It is probably easier to setup for you.

**Comment by nicolas33** [comment]
Yes, I've seen issues with travis. It really looks like the tool isn't anymore what it used to be. I'd move to something else. The choice for Travis was years ago and a lot of things have change since then.

**Comment by chris001** [comment]
@nicolas33 @thekix  I've sent you email about this, did you receive?

**Comment by thekix** [comment]
Yes!!

Thanks! I will reply this weekend.

**Comment by thekix** [comment]
Added to branch testing.

Thanks
kix
